cmd/snap/cmd_auto_import.go

```diff
@@ -92,6 +94,38 @@ func autoImportCandidates() ([]string, error) {
 
 }
 
+func queueFile(src string) error {
+	dst := filepath.Join(dirs.SnapAssertsSpoolDir, strutil.MakeRandomString(16))
+	if err := os.MkdirAll(filepath.Dir(dst), 0755); err != nil {
+		return err
+	}
+	return osutil.CopyFile(src, dst, 0)
+}
+
+func autoImportFromSpool() (added int, err error) {
+	files, err := ioutil.ReadDir(dirs.SnapAssertsSpoolDir)
+	if os.IsNotExist(err) {
+		return 0, nil
+	}
+	if err != nil {
+		return added, err
+	}
+	for _, fi := range files {
+		cand := filepath.Join(dirs.SnapAssertsSpoolDir, fi.Name())
+		if err := ackFile(cand); err != nil {
```

Comment:
Should we have a look into the file stat data, and remove it after a while to prevent this from growing indefinitely when a particular file is not understood?